client/src/extension.ts

```diff
@@ -89,8 +90,26 @@ export function activate(context: ExtensionContext) {
         ExecYseopCliCommand(yseopCliPath, 'package');
     });
 
-    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.info`, () => {
-        ExecYseopCliCommand(yseopCliPath, 'info');
+    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.config`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'config');
+    });
+
+    const libsInstallCmd = commands.registerCommand(`${yseopmlSectionName}.libs`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'libs', 'install', '-R');
+    });
+
+    const deployCmd = commands.registerCommand(`${yseopmlSectionName}.managerDeploy`, (ymaUri: Uri) => {
+        window
+            .showInputBox({ prompt: 'Leave this box empty to use the default env.', placeHolder: 'DEMO' })
```

Comment:
* env → environment
* Having “DEMO” as an hardcoded placeholder seems dangerous to me. This gives the impression that it is the “default env” you’re talking about. Either make the default configurable and align the placeholder with it, or leave it empty, because the default env does not use any “key string”: it’s just `YSEOP_CLI_MANAGER_URL/USERNAME/PASSWORD` stuff.

Reply:
OK!